Consul/Operator.cs

```diff
@@ -229,6 +229,44 @@ public Task<WriteResult> KeyringUse(string key, WriteOptions q, CancellationToke
         {
             return _client.Put("/v1/operator/keyring", new KeyringRequest() { Key = key }, q).Execute(ct);
         }
+
+        public Task<QueryResult<ConsulLicense>> GetConsulLicense(string datacenter = "", CancellationToken ct = default)
```

Comment:
I think it would be good to add a comment explaining that this is enterprise-only feature